src/components/button/Button.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import styles from './Button.css';
+
+export default class Button extends Component {
```

Comment:
Same as above:

> After talking with @satazor we will schedule a meeting to discuss Functional vs Stateful Components.

Reply:
As of this time, the button will remain a class component. It will be changed to a functional component when TooltipTrigger receives the changes requested in https://github.com/ipfs-shipyard/pm-idm/issues/78 .